auspice-client/customisations/splash.js

```diff
@@ -114,17 +114,19 @@ function ErrorMessage({errorMessage}) {
   );
 }
 
+const ListItem = styled.li`
+  margin: 5px 0;
+`
+
 /* lifted from auspice's default splash page */
-function formatDataset(requestPath, dispatch, changePage) {
+function formatDataset(requestPath) {
+
+  // FIXME: Do we actually need dispatch and changePage here? I've removed them for now.
```

Comment:
This is my main concern - it's not clear to me what `dispatch` and `changePage` were used for. Seems to work fine without it.

Reply:
The behaviour will be a little different -- dispatching stays within the SPA, whereas links will go to a new URL, fetch the auspice HTML entrypoint, fetch the auspice JS (cache hit), reparse the JS and so on. In practice, the Auspice splash page within nextstrain.org is used rarely - perhaps only for community data?

P.S. I've wanted proper link behaviour for a while now, especially in the default Auspice splash page.